src/st/no3/377D4.c

```diff
@@ -175,7 +205,32 @@ INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801BD71C);
 
 INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801BDEB0);
 
-INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801BE080);
+extern Entity D_8007C0D8[];
```

Comment:
Can you move this one into `game.h`? Everything from `80010000` up to `800A0000` goes under `game.h`, which is the shared memory area across all the overlays. So it makes sense to not declare it exclusively inside the overlay code.